dist/newrelic/newrelic.yml

```diff
@@ -0,0 +1,302 @@
+# This file configures the New Relic Agent.  New Relic monitors
```

Comment:
I believe the one you are referencing is for heroku. I'm not sure where it came from but I think it's just from an older version of the new relic agent.   This one is for the upgraded new relic agent (ver 3.39.1) and it's placed in the dist folder so that it will get deployed to the elastic beanstalk ec2 instances.    I guess we can attempt to consolidate them but I didn't feel the need to do that since the heroku one can will just go away once we move to AWS.

In terms of setting configs with newrelic.yml vs from web ui.  There shouldn't be any problems setting configs in yml and web ui.  The web ui settings take precedence over yml file configs[1] allowing you to override the configs.   I actually prefer setting the configs in yml so that it's under source control.

[1] https://docs.newrelic.com/docs/agents/java-agent/configuration/java-agent-configuration-config-file